src/System.Management.Automation/engine/parser/tokenizer.cs

```diff
@@ -1551,7 +1551,7 @@ private void ScanToEndOfCommentLine(out bool sawBeginSig, out bool matchedRequir
 
         #endregion Utilities
 
-        #region Object reuse
+        #region object reuse
```

Comment:
Is there a convention here?  Seems like the uppercase is more consistent with usage in the source code and slightly easier to read

Reply:
No convention exists. Reverted for better readability.